packages/xo-server/src/api/vm.js

```diff
@@ -1144,7 +1147,7 @@ revert.params = {
 }
 
 revert.resolve = {
-  snapshot: ['snapshot', 'VM-snapshot', 'administrate'],
+  snapshot: ['snapshot', 'VM-snapshot', false],
```

Comment:
`false` → `'view'`

Reply:
Any vm operator is considered as `viewer` of `snapshot` no ?

Reply:
We've discussed about this.
A user has to be viewer of snapshot and operator of VM in order to revert it